### PR TITLE
Disaster recovery with snapshots

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: content/vendor/snapshots/disaster-recovery-snapshots.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Restore must begin with installing a version of Velero compatible with the one that was used to make the snapshot.
+Please refer to Velero documnetion for [installing](https://velero.io/docs/v1.5/basic-install/) and [configuring](https://velero.io/plugins/) the plugins.
+Note that Restic is required and `--restic` flag must be used with `velero install` command.
+
+Once Velero is installed, KOTS CLI can be used to [list backups](/kots-cli/backup-ls/) and [create restores](/kots-cli/restore/)
+
+## Online Embedded Cluster Restore
+
+The procedure for restoring an online embedded cluster is the same as the one for restoring an existing cluster.
```
Comment (salahalsaleh) [inline content/vendor/snapshots/disaster-recovery-snapshots.md:22]: should we mention that when restoring to a clean instance, the installer will have to be run first?

Comment (salahalsaleh) [reply content/vendor/snapshots/disaster-recovery-snapshots.md:22]: also, there's no doc for restoring in an existing cluster? should we link to the restore command?

Comment (divolgin) [reply content/vendor/snapshots/disaster-recovery-snapshots.md:22]: @salahalsaleh it's the section right above.  Maybe it needs more detail?

**Review by salahalsaleh — APPROVED** [review]

File: content/vendor/snapshots/disaster-recovery-snapshots.md (hunk 1 above)
Comment (salahalsaleh) [reply content/vendor/snapshots/disaster-recovery-snapshots.md]: yes, my bad

**Pull request merged**